client/my-sites/stats/stats-list/style.scss

```
@@ -542,16 +542,16 @@ ul.module-content-list-item-action-submenu {
 	}
 
 	// Color follow action when already following
-	.module-content-list-item-action-wrapper.following .module-content-list-item-action-label {
-		color: var(--color-success);
+	.module-content-list-item-action-wrapper.following .module-content-list-item-action-label,
+	.module-content-list-item-action-wrapper.following .module-content-list-item-action-label:hover {
+		color: var(--studio-white);
```

You can avoid repeating the long classnames here by using the parent selector `&:hover`. You do need to repeat the color declaration instead, but it's potentially neater?

```suggestion
	.module-content-list-item-action-wrapper.following .module-content-list-item-action-label {
		color: var(--studio-white);
		
		&:hover {
		  color: var(--studio-white);
		}
```

---

Ah nice, I do think it's a little neater. Thank you!